### DISCUSSION TITLE
Followup for multiple map loading

### DISCUSSION THREAD

**Discussion by parasyte** [melonjs/melonJS#1046] [Ideas]
Follows #704 

This is totally broken. :smiley: I recorded my entire attempt to get this working: http://www.twitch.tv/kodewerx/v/18891201 The fun with multiple maps starts at about 01:21:00

(NOTE: This video will not be on Twitch forever! I'll upload it to YouTube at some point. At least the relevant part.)
- The current map disappears
- The next map sometimes doesn't render
- The next map replaces the "current level" reference
- The map location bounces all over the place (when smaller than the viewport)

What a mess. :)

**Comment by obiot** [comment]
about the "currentLevel" issue, indeed, the whole API is made around the "one level loaded at a time" idea. so even all the function in the LevelDirector are to be changed to break the 1:1 relationship between level and TMX file

**Comment by obiot** [comment]
-> 3.1.0
